spec.html

```diff
@@ -37,6 +37,151 @@ <h1>Introduction</h1>
   <p>`Promise.any()` accepts an array of promises and returns a promise that is fulfilled by the first given promise to be fulfilled, or rejected with an array of rejection reasons if all of the given promises are rejected.</p>
 </emu-intro>
 
+<emu-clause id="sec-well-known-intrinsic-objects">
+  <h1>Well-Known Intrinsic Objects</h1>
+  <p>Well-known intrinsics are built-in objects that are explicitly referenced by the algorithms of this specification and which usually have realm-specific identities. Unless otherwise specified each intrinsic object actually corresponds to a set of similar objects, one per realm.</p>
+  <p>Within this specification a reference such as %name% means the intrinsic object, associated with the current realm, corresponding to the name. Determination of the current realm and its intrinsics is described in <emu-xref href="#sec-execution-contexts"></emu-xref>. The well-known intrinsics are listed in <emu-xref href="#table-7"></emu-xref>.</p>
+  <emu-table id="table-7" caption="Well-Known Intrinsic Objects">
+    <table>
+      <tbody>
+      <tr>
+        <th>
+          Intrinsic Name
+        </th>
+        <th>
+          Global Name
+        </th>
+        <th>
+          ECMAScript Language Association
+        </th>
+      </tr>
+      <tr>
+        <td>
+          <ins>%AggregateError%</ins>
+        </td>
+        <td>
+          <ins>`AggregateError`</ins>
+        </td>
+        <td>
+          <ins>The `AggregateError` constructor (<emu-xref href="#sec-aggregate-error-constructor"></emu-xref>)</ins>
+        </td>
+      </tr>
+      <tr>
+        <td>
+          <ins>%AggregateError.prototype%</ins>
+        </td>
+        <td>
+          <ins>`AggregateError.prototype`</ins>
+        </td>
+        <td>
+          <ins>The initial value of the `prototype` data property of %AggregateError%</ins>
```

Comment:
i don’t think this item needs to be added; as long as the constructor is added.

However, due to the NativeErrorPrototype stuff, i think we *do* need a `%AggregateErrorPrototype%` intrinsic.

Reply:
`AggregateError` has different constructor behavior than `NativeError`s as @bakkot mention in #27

Reply:
All `NativeError`s have this

Reply:
Right, but with the old dotless intrinsic style, which is what I’m suggesting is needed here.